### PR TITLE
Fixed typo of assign-pod-node.md

### PR DESCRIPTION
**Pull request by szhnet**
Fixed typo of assign-pod-node.md

### CODE REVIEW

**Comment by szhnet** [comment]
I signed CLA

File: content/zh/docs/concepts/configuration/assign-pod-node.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ that a pod ends up on a machine with an SSD attached to it, or to co-locate pods
 services that communicate a lot into the same availability zone.
 -->
```
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/assign-pod-node.md:33]: see https://identity.linuxfoundation.org/projects/cncf

```

If you haven't yet, post a comment ("I signed it") to the PR—this is needed for the system to recheck your contribution authorization.
```

**Comment by szhnet** [comment]
I signed it

**Comment by szhnet** [comment]
I signed it

**Pull request merged**